nasa/openmct
# Fix foreground color of option elements in selects

### Changes
Windows users in Snow theme reporting white foreground color on system background color (light gray) when interacting with select elements. This fix hard-codes the option's `color` attribute to black.

### Verification

1. Run `topic-core-refactor` in Windows Chrome with the Snow theme and interact with a custom select, such as make a new Timer and interact with the Display Format menu, or editing a Summary Widget's condition.
1. Observe: the system-based options menu should display light text on a light neutral background and be hard to read.
1. Run this branch `fix-selects` in the same system\browser\theme and perform the same test as 2.
1. Observe: the options menu should present readable black text on a light neutral background.

### Author Checklist
1. Changes address original issue? Y
1. Unit tests included and/or updated with changes? N/A
1. Command line build passes? Y
1. Changes have been smoke-tested? N: needs environ as described above.
